{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/static_source/css/components/components.scss

```diff
@@ -0,0 +1,2 @@
+@import "@/../templates/components/links/links.scss";
```

Comment:
I think this is a no op, since they're imported by the components?

Reply:
right I've removed this line here